CHANGELOG.md

```diff
@@ -35,14 +36,13 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 
 - Remove `flow.id` and `task.id` attributes - [#940](https://github.com/PrefectHQ/prefect/pull/940)
 - Removed old WIP environments - [#936](https://github.com/PrefectHQ/prefect/pull/936)
-*Note*: Changes from [#936](https://github.com/PrefectHQ/prefect/pull/936) regarding environments don't break any Prefect code because environments weren't used yet outside of Cloud.
+  _Note_: Changes from [#936](https://github.com/PrefectHQ/prefect/pull/936) regarding environments don't break any Prefect code because environments weren't used yet outside of Cloud.
```

Comment:
```suggestion
  (_Note_: Changes from [#936](https://github.com/PrefectHQ/prefect/pull/936) regarding environments don't break any Prefect code because environments weren't used yet outside of Cloud.)
```